contexts/retail-coupon/v1.json

```
@@ -0,0 +1,116 @@
+{
+  "@context": {
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+    "gs1": "https://gs1.org/voc/",
+
+    "ClippedCouponCredential": {
+      "@id": "https://contexts.vcplayground.org/examples/retail-coupon/vocab#ClippedCouponCredential"
+    },
```

All the places where it's only defining the id of a term can be simplified to one-liners.
```suggestion
    "ClippedCouponCredential": "https://contexts.vcplayground.org/examples/retail-coupon/vocab#ClippedCouponCredential",
```